src/answers-umd.js

```diff
@@ -297,7 +297,11 @@ class Answers {
 
     this._setDefaultInitialSearch(parsedConfig.search);
 
-    this.core.init();
+    if (parsedConfig.visitor) {
+      this.setVisitor(parsedConfig.visitor);
```

Comment:
maybe it would make sense for core.init to take in a visitor param?

Reply:
updated!